src/app/tests/suites/certification/Test_TC_BOOL_1_1.yaml

```diff
@@ -34,39 +34,67 @@ tests:
       response:
           value: 1
 
-    - label: "Read the global attribute constraints: ClusterRevision"
+    - label: "Read the global attribute constraints : ClusterRevision"
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
           constraints:
               type: uint16
 
+    #issue #17480
```

Comment:
This test is wrong; see the (resolved) issue.

Reply:
@bzbarsky-apple i will update this script